### PR TITLE
Removed a content

### PR DESCRIPTION
**Pull request by joinimran**
There is no need to create a custom device policy to enable PIN reset recovery. So removing this portion.

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/8284#issuecomment-707356130

### CODE REVIEW

**Comment by joinimran** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/identity-protection/hello-for-business/hello-feature-pin-reset.md [hunk 1]
```diff
@@ -66,9 +66,6 @@ You configure Windows 10 to use the Microsoft PIN Reset service using the comput
 3. Enable the **Use PIN Recovery** policy setting located under **Computer Configuration->Administrative Templates->Windows Components->Windows Hello for Business**.
 4. Close the Group Policy Management Editor to save the Group Policy object.  Close the GPMC.
 
-### Configure Windows devices to use PIN reset using Microsoft Intune
-
-To configure PIN reset on Windows devices you manage, use an [Intune Windows 10 custom device policy](https://docs.microsoft.com/intune/custom-settings-windows-10) to enable the feature. Configure the policy using the following Windows policy configuration service provider (CSP):
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-feature-pin-reset.md:69]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by joinimran** [comment]
@e0i - Ready to go live.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by illfated** [comment]
@joinimran : Thank you for the work you provide in pull requests and ticket comments & replies. Just a minor detail here in this PR ("English grammar tip of the day", if you like): the word "content" is usually considered to describe some undefined amount of mass or uncountable number of items. The most common usage is to skip "a", going straight to the word itself, so in this case the PR title would in most cases be "Removed content" (or "Removed outdated content").

Anyway, this is mostly an advice for future use. No need to change the PR title, because the commit name will stay unchanged.

**Review by illfated — APPROVED** [review]
Looks good & correct to me.

**Comment by illfated** [comment]
#sign-off

**Comment by mapalko** [comment]
Instead of removing this section, it should be replaced with steps for configuring the policy via the UI linked by CHDAFNI-MSFT

**Comment by e0i** [comment]
> Instead of removing this section, it should be replaced with steps for configuring the policy via the UI linked by CHDAFNI-MSFT

cc @joinimran

**Comment by joinimran** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/identity-protection/hello-for-business/hello-feature-pin-reset.md [hunk 2]
```diff
@@ -66,26 +66,12 @@ You configure Windows 10 to use the Microsoft PIN Reset service using the comput
...
 
-1. Sign-in to [Azure Portal](https://portal.azure.com) using a Global administrator account.
-2. You need your tenant ID to complete the following task.  You can discover your tenant ID by viewing the **Properties** of your Azure Active Directory from the Azure Portal. It will be listed under Directory ID. You can also use the following command in a Command window on any Azure AD-joined or hybrid Azure AD-joined computer.</br>
-
-    ```
-    dsregcmd /status | findstr -snip "tenantid"
-    ```
+1. Sign-in to [Enpoint Manager Admin Center](https://endpoint.microsoft.com/) using a Global administrator account.
+2. Click **Endpoint Security**-> **Account Portection**-> **Properties**.
+3. Select Enable PIN recovery to **Yes**.
```
Comment (illfated) [inline windows/security/identity-protection/hello-for-business/hello-feature-pin-reset.md:73]: I think "Set" is better than "Select" in this context, but maybe it needs further rephrasing.
```suggestion
3. Set **Enable PIN recovery** to **Yes**.
```
@JohanFreelancer9 , can you confirm or suggest an improvement?

**Comment by joinimran** [comment]
Thanks, @illfated. This makes more sense.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by illfated — APPROVED** [review]
Looks good to me.

File: windows/security/identity-protection/hello-for-business/hello-feature-pin-reset.md [hunk 3]
```diff
@@ -66,26 +66,12 @@ You configure Windows 10 to use the Microsoft PIN Reset service using the comput
...
 #### Create a PIN Reset Device configuration profile using Microsoft Intune
 
-1. Sign-in to [Azure Portal](https://portal.azure.com) using a Global administrator account.
-2. You need your tenant ID to complete the following task.  You can discover your tenant ID by viewing the **Properties** of your Azure Active Directory from the Azure Portal. It will be listed under Directory ID. You can also use the following command in a Command window on any Azure AD-joined or hybrid Azure AD-joined computer.</br>
-
-    ```
-    dsregcmd /status | findstr -snip "tenantid"
-    ```
+1. Sign-in to [Enpoint Manager Admin Center](https://endpoint.microsoft.com/) using a Global administrator account.
+2. Click **Endpoint Security**-> **Account Portection**-> **Properties**.
```
Comment (mapalko) [inline windows/security/identity-protection/hello-for-business/hello-feature-pin-reset.md:72]: Typo: "Account Portection" should be "Account Protection"

File: windows/security/identity-protection/hello-for-business/hello-feature-pin-reset.md [hunk 4]
```diff
@@ -66,26 +66,12 @@ You configure Windows 10 to use the Microsoft PIN Reset service using the comput
...
+3. Set **Enable PIN recovery** to **Yes**.
```
Comment (mapalko) [inline windows/security/identity-protection/hello-for-business/hello-feature-pin-reset.md:73]: Endpoint Security/Account Protection is an optional configuration and this step assumes someone has already set it up.

Can we also add an alternative set of steps using configuration profiles?
1. Sign-in to Endpoint Manager
2. Click **Devices** -> **Configuration Profiles** -> Create new profile or edit existing an existing profile using the Identity Protection profile type 
Set **Enable PIN recovery** to **Yes**

**Review by mapalko — CHANGES_REQUESTED** [review]

File: windows/security/identity-protection/hello-for-business/hello-feature-pin-reset.md [hunk 5]
```diff
@@ -66,26 +66,12 @@ You configure Windows 10 to use the Microsoft PIN Reset service using the comput
...
-
 #### Create a PIN Reset Device configuration profile using Microsoft Intune
 
-1. Sign-in to [Azure Portal](https://portal.azure.com) using a Global administrator account.
-2. You need your tenant ID to complete the following task.  You can discover your tenant ID by viewing the **Properties** of your Azure Active Directory from the Azure Portal. It will be listed under Directory ID. You can also use the following command in a Command window on any Azure AD-joined or hybrid Azure AD-joined computer.</br>
-
-    ```
-    dsregcmd /status | findstr -snip "tenantid"
-    ```
+1. Sign-in to [Enpoint Manager Admin Center](https://endpoint.microsoft.com/) using a Global administrator account.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-feature-pin-reset.md:71]:
```suggestion
1. Sign-in to [Enpoint Manager admin center](https://endpoint.microsoft.com/) using a Global administrator account.
```

File: windows/security/identity-protection/hello-for-business/hello-feature-pin-reset.md (hunk 3 above)
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-feature-pin-reset.md:72]:
```suggestion
2. Click **Endpoint Security**-> **Account Protection**-> **Properties**.
```

**Comment by joinimran** [comment]
@JohanFreelancer9 please copy review.

**Review by mapalko — APPROVED** [review]

File: windows/security/identity-protection/hello-for-business/hello-feature-pin-reset.md [hunk 6]
```diff
@@ -66,26 +66,18 @@ You configure Windows 10 to use the Microsoft PIN Reset service using the comput
...
-1. Sign-in to [Azure Portal](https://portal.azure.com) using a Global administrator account.
-2. You need your tenant ID to complete the following task.  You can discover your tenant ID by viewing the **Properties** of your Azure Active Directory from the Azure Portal. It will be listed under Directory ID. You can also use the following command in a Command window on any Azure AD-joined or hybrid Azure AD-joined computer.</br>
+1. Sign-in to [Enpoint Manager admin center](https://endpoint.microsoft.com/) using a Global administrator account.
+2. Click **Endpoint Security**-> **Account Protection**-> **Properties**.
+3. Set **Enable PIN recovery** to **Yes**.
 
-    ```
-    dsregcmd /status | findstr -snip "tenantid"
-    ```
+> [!Note]
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-feature-pin-reset.md:75]:
```suggestion
> [!NOTE]
```

File: windows/security/identity-protection/hello-for-business/hello-feature-pin-reset.md [hunk 7]
```diff
@@ -66,26 +66,18 @@ You configure Windows 10 to use the Microsoft PIN Reset service using the comput
...
+> You can also setup PIN recovery using configuration profiles.
+> 1. Sign-in to Endpoint Manager
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-feature-pin-reset.md:77]:
```suggestion
> 1. Sign in to Endpoint Manager.
```

File: windows/security/identity-protection/hello-for-business/hello-feature-pin-reset.md [hunk 8]
```diff
@@ -66,26 +66,18 @@ You configure Windows 10 to use the Microsoft PIN Reset service using the comput
...
+> 2. Click **Devices** -> **Configuration Profiles** -> Create new profile or edit existing an existing profile using the Identity Protection profile type
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-feature-pin-reset.md:78]:
```suggestion
> 2. Click **Devices** -> **Configuration Profiles** -> Create new profile or edit an existing profile using the Identity Protection profile type.
```

File: windows/security/identity-protection/hello-for-business/hello-feature-pin-reset.md [hunk 9]
```diff
@@ -66,26 +66,18 @@ You configure Windows 10 to use the Microsoft PIN Reset service using the comput
...
+> 3. Set **Enable PIN recovery** to **Yes**
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-feature-pin-reset.md:79]:
```suggestion
> 3. Set **Enable PIN recovery** to **Yes**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by joinimran** [comment]
Thanks, @JohanFreelancer9.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by illfated — APPROVED** [review]

**Comment by illfated** [comment]
Link to rich diff view of the image:
- https://github.com/MicrosoftDocs/windows-itpro-docs/pull/8457/files?short_path=d82adde#diff-d82addeadf260315de8dde6cd3b4d68258bc3ab9865e09ca461d779301e141dc 

(pick Swipe or Onion Skin to get a big enough image and a small tug handle at the lower left corner to view the text change in the box (in the middle of the image) when moving the handle)

**Review by illfated — APPROVED** [review]
Looks quite good to me.

**Comment by illfated** [comment]
(also ref. #8509 )

**Comment by illfated** [comment]
#sign-off

**Comment by joinimran** [comment]
> Oh. Is the issue that this MacOS image does not belong in this PR, [MicrosoftDocs#8457](https://github.com/MicrosoftDocs/windows-itpro-docs/pull/8457) , which is about the Hello for Business _Pin Reset_ feature?

**Comment by joinimran** [comment]
> Oh. Is the issue that this MacOS image does not belong in this PR, [MicrosoftDocs#8457](https://github.com/MicrosoftDocs/windows-itpro-docs/pull/8457) , which is about the Hello for Business _Pin Reset_ feature?

@illfated - That looks like for a different page and different content.

**Pull request closed without merging**